test/bdd/pkg/v1/vc/credential.go

```diff
@@ -289,6 +335,27 @@ func (e *Steps) checkVC(vcBytes []byte, profileName, signatureRepresentation str
 	return nil
 }
 
+func (e *Steps) checkIssuedSDJWTVC(vcBytes []byte) error {
+	b := vcsverifiable.UnQuote(vcBytes)
+	if !vcsverifiable.IsSDJWT(string(b)) {
+		return fmt.Errorf("issued VC %s is not SD-JWT", string(vcBytes))
+	}
+
+	cfi := common.ParseCombinedFormatForIssuance(string(b))
```

Comment:
Use holder.Parse() function to check validity of combined format for issuance. That function will raise an exception if SD-JWT parsing fails or if there are disclosures that are not part of SD-JWT.

Reply:
By default it will not check signature. If you would like to check signature than you need to pass in verifier.

Reply:
good catch, thanks !